app/controllers/teacher_sets_controller.rb

```diff
@@ -77,8 +77,8 @@ def show
 
     # limits book titles to unique ones to mask the problem we've been having
     # with the teacher_sets detail page puling up duplicate child book records.
-    ts_books = @set.books.select(:cover_uri).distinct
-
+    ts_books = @set.books.select('DISTINCT ON (books.cover_uri) books.cover_uri, books.id, books.title')
```

Comment:
Confused why this is in this PR.

Reply:
Is this related to MLN-550? Maybe add some explanation in these comments as to why. We can chat tomorrow in stand-up too.